<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebitNoteForm</class>
 <widget class="QWidget" name="DebitNoteForm">
  <property name="geometry"><rect><x>0</x><y>0</y><width>1200</width><height>900</height></rect></property>
  <property name="windowTitle"><string>DebitNoteForm</string></property>
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <property name="spacing"><number>8</number></property>
   <property name="margin"><number>8</number></property>

   <!-- Top -->
   <item>
    <widget class="QFrame" name="frameTopInfo">
     <property name="frameShape"><enum>QFrame::StyledPanel</enum></property>
     <layout class="QHBoxLayout" name="horizontalLayout_topinfo">
      <property name="spacing"><number>10</number></property>
      <property name="margin"><number>8</number></property>
      <item><widget class="QLabel" name="labelLogo"><property name="minimumSize"><size><width>140</width><height>90</height></size></property><property name="frameShape"><enum>QFrame::Box</enum></property><property name="text"><string>LOGO</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
      <item>
       <widget class="QFrame" name="frameTitle">
        <layout class="QVBoxLayout" name="verticalLayout_title">
         <item><widget class="QLabel" name="labelTitle"><property name="text"><string>DEBIT NOTE</string></property><property name="font"><font><pointsize>18</pointsize><weight>75</weight><bold>true</bold></font></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
         <item><widget class="QLabel" name="labelSubTitle"><property name="text"><string>Debit Note Generator</string></property><property name="alignment"><set>Qt::AlignCenter</set></property></widget></item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameInvoiceMeta">
        <layout class="QFormLayout" name="formLayout_meta">
         <item row="0" column="0"><widget class="QLabel" name="labelInvNoMeta"><property name="text"><string>Debit Note No:</string></property></widget></item>
         <item row="0" column="1"><widget class="QLineEdit" name="leDebitNo"/></item>
         <item row="1" column="0"><widget class="QLabel" name="labelDateMeta"><property name="text"><string>Date:</string></property></widget></item>
         <item row="1" column="1"><widget class="QLineEdit" name="leDate"/></item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Main left/right -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_mainSections">
     <property name="spacing"><number>12</number></property>

     <!-- left -->
     <item>
      <widget class="QFrame" name="frameLeftColumn">
       <layout class="QVBoxLayout" name="verticalLayout_left">
        <item>
         <layout class="QHBoxLayout" name="hl_customer">
          <item><widget class="QLabel" name="labelCustomer"><property name="text"><string>Customer:</string></property></widget></item>
          <item><widget class="QComboBox" name="cbCustomer"/></item>
          <item><widget class="QPushButton" name="btnAddCustomer"><property name="text"><string>+ Add</string></property></widget></item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hl_address">
          <item><widget class="QLabel" name="labelAddressSelect"><property name="text"><string>Address:</string></property></widget></item>
          <item><widget class="QComboBox" name="cbAddress"/></item>
         </layout>
        </item>

        <item><widget class="QLabel" name="labelBillToTitle"><property name="text"><string>BILL TO:</string></property><property name="font"><font><weight>75</weight><bold>true</bold></font></property></widget></item>
        <item><widget class="QTextEdit" name="teBillTo"/></item>

        <item><widget class="QLabel" name="labelShippingTitle"><property name="text"><string>SHIPMENT DETAILS</string></property></widget></item>

        <item>
         <widget class="QFrame" name="frameShipping">
          <layout class="QFormLayout" name="formLayout_shipping">
           <item row="0" column="0"><widget class="QLabel" name="labelShipper"><property name="text"><string>Shipper:</string></property></widget></item>
           <item row="0" column="1"><widget class="QLineEdit" name="leShipper"/></item>
           <item row="1" column="0"><widget class="QLabel" name="labelShipConsignee"><property name="text"><string>Consignee:</string></property></widget></item>
           <item row="1" column="1"><widget class="QLineEdit" name="leShipConsignee"/></item>
           <item row="2" column="0"><widget class="QLabel" name="labelPOL"><property name="text"><string>POL:</string></property></widget></item>
           <item row="2" column="1"><widget class="QLineEdit" name="lePOL"/></item>
           <item row="3" column="0"><widget class="QLabel" name="labelPOD"><property name="text"><string>POD:</string></property></widget></item>
           <item row="3" column="1"><widget class="QLineEdit" name="lePOD"/></item>
           <item row="4" column="0"><widget class="QLabel" name="labelVessel"><property name="text"><string>Vessel/Flight:</string></property></widget></item>
           <item row="4" column="1"><widget class="QLineEdit" name="leVessel"/></item>
           <item row="5" column="0"><widget class="QLabel" name="labelETD"><property name="text"><string>ETD:</string></property></widget></item>
           <item row="5" column="1"><widget class="QLineEdit" name="leETD"/></item>
           <item row="6" column="0"><widget class="QLabel" name="labelETA"><property name="text"><string>ETA:</string></property></widget></item>
           <item row="6" column="1"><widget class="QLineEdit" name="leETA"/></item>
          </layout>
         </widget>
        </item>

        <item><widget class="QLabel" name="labelConsigneePreview"><property name="text"><string>Consignee (preview):</string></property></widget></item>
        <item><widget class="QTextEdit" name="teConsignee"/></item>

       </layout>
      </widget>
     </item>

     <!-- right: items table (same as invoice) -->
     <item>
      <widget class="QFrame" name="frameRightColumn2">
       <layout class="QVBoxLayout" name="verticalLayout_tableArea">
        <item>
         <widget class="QTableWidget" name="tableItems">
          <property name="columnCount"><number>13</number></property>
          <property name="rowCount"><number>0</number></property>
          <!-- columns omitted for brevity (same as invoice UI) -->
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frameTableButtons">
          <layout class="QHBoxLayout" name="horizontalLayout_tableButtons">
           <item><widget class="QPushButton" name="btnAddRow"><property name="text"><string>Add Row</string></property></widget></item>
           <item><widget class="QPushButton" name="btnDelRow"><property name="text"><string>Delete Row</string></property></widget></item>
           <item><spacer name="spacer_tableButtons"><property name="sizeHint"><size><width>40</width><height>20</height></size></property><property name="sizePolicy"><sizepolicy><hor>QSizePolicy.Expanding</hor><ver>QSizePolicy.Fixed</ver></sizepolicy></property></spacer></item>
           <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Save Debit Note</string></property></widget></item>
           <item><widget class="QPushButton" name="btnExportPDF"><property name="text"><string>Export PDF</string></property></widget></item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>

    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
